src/app/components/media/MediaThumbnail.tsx

```diff
@@ -23,6 +23,12 @@ const FALLBACK_EXTENSIONS = [
   "docx"
 ];
 
+export const THUMBNAIL_VIEW_MODE_KEY = "thumbnailMode";
+
+export type ThumbnailViewMode =
+  | "image" // only thumbnail
+  | "text"; // fallback-icon and fallback-label
```

Comment:
[minor] Ich würde die noch umbenennen zu sowas wie "image-only" und "text-fallback"; dann haben wir später auch die Möglichkeit, da noch weitere Modi wie "image-and-text" o.ä. zu setzen…